### ISSUE TITLE
eval.py报错

### ISSUE THREAD

**Issue by LinJM** [UpCoder/ICPR_TextDection#2]
NotFoundError (see above for traceback): Key feature_fusion/Conv_10/BatchNorm/beta/ExponentialMovingAverage not found in checkpoint
	 [[Node: save/RestoreV2 = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2/tensor_names, save/RestoreV2/shape_and_slices)]]
	 [[Node: save/RestoreV2/_301 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device_incarnation=1, tensor_name="edge_306_save/RestoreV2", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:GPU:0"]()]]

**Comment by UpCoder** [comment]
Did you check the path of checkpoint file? Is it correct? It seem that the tensor isn't found in the pre-trained model.

**Comment by LinJM** [comment]
I use the checkpoint from your Trained model (Models trained on ICDAR 2013 (training set) + ICDAR 2015 (training set): GoogleDrive)

**Comment by UpCoder** [comment]
Hi, I updated README.md. You can download the trained model from https://pan.baidu.com/s/1WK2GdnXpJZFmgFe5uh0tng. The trained model that you download is the original version, you can see the original code in [here](https://github.com/argman/EAST)

**Comment by LinJM** [comment]
Thanks
